spec/fixtures/ssl/ec-key-openssl.pem

```diff
@@ -0,0 +1,5 @@
+-----BEGIN PRIVATE KEY-----
```

Comment:
Should this be a fixture with the extra params at the top?

Reply:
If I take a key that has them and run it through the `-topk8`, it removes the extra params...